controllers/testimonialsController.js

```diff
@@ -0,0 +1,11 @@
+import { testimonialsList } from "../services/testimonialsServices.js";
+
+export const getAllTestimonials = async (req, res, next) => {
```

Comment:
Instead of try/catch block here you can use ctrlWrappe method from decorators: 
export default {
    getAllTestimonials: ctrlWrappe(getAllTestimonials)
}

Reply:
Corrected